packages/showcases/react/stories/components/structure/VtmnAccordion/VtmnAccordion.stories.tsx

```diff
@@ -59,7 +59,13 @@ const WithLeftIconsTemplate: Story = (args) => (
 
 const CustomContentTemplate: Story = (args) => (
   <div style={{ width: 400, height: 250 }}>
-    <VtmnAccordion summary="Item 1" withIconLeft open>
+    <VtmnAccordion
+      summary="Item 1"
+      withIconLeft
+      open
+      onOpen={(e) => console.log('onOpen callback', e)}
+      onClose={(e) => console.log('onClose callback', e)}
```

Comment:
Can you please use the `@storybook/addon-actions`?
Btw, I don't see any `alert` when I've tried to launch your storybook.